protoc_plugin/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.13.0
+
+* Breaking change: generated message classes now have a new instance method `createNewInstance` that
```

Comment:
createEmptyInstance ?

Reply:
Done

Reply:
no...

Reply:
Please try to avoid adding instance methods.

I would like to do a compiler optimization where FooMessage is replaced with GeneratedMessage unless there is a test like 'is FooMessage'.
This will remove many of the thousands of generated classes, but it is only possible if the generated message classes have instance methods that are always inlined (the field get/set/has methods are resolved to one target and inlined, leaving the class 'empty').
createNewInstance is defined in the base class and overridden so is not inlined.

This can be fixed by storing FooMessage.create in the BuilderInfo, and using it from there.

Reply:
Sad that I got to release a version before seeing your comment. :(

We have the same problem with `clone()` no?

Could we store the `FooMessage.create()` method in `BuilderInfo` as you suggest, and then make this method non-abstract and use that?